config.go

```diff
@@ -30,8 +31,9 @@ const (
 //	})
 //
 type FileConfig struct {
-	SkipPageIndex    bool
-	SkipBloomFilters bool
+	SkipPageIndex       bool
+	SkipBloomFilters    bool
+	GetIOReaderFromPath func(filepath string) io.ReaderAt
```

Comment:
What do you think about using an interface type here instead of a function?

Reply:
Done, but I'm not sure about what the initial signature should look like. The application should have enough information about the object to provide a secondary reader, so I'm not even sure we need to pass the column chunk metadata